README.md

```diff
@@ -1,127 +1,98 @@
-
-2. The repo owners will respond to your issue promptly.
-
-3. If your proposed change is accepted, sign the [CLA](https://cla-assistant.io/pingcap/tidb?pullRequest=5567), and start work in your fork.
-
-4. Submit a [pull request](https://github.com/pingcap/tidb/pull/3113) containing a tested change. 
-
-
-Once you become a TiDB contributor, check your name here: [https://github.com/pingcap/tidb/blob/master/CONTRIBUTORS](https://github.com/pingcap/tidb/blob/master/CONTRIBUTORS)
-
-Contributions are welcomed and greatly appreciated. See [CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
-
-## How Do I Become a Committer?
-
-First of all you need to get involved and be a Contributor.
-
-Based on your track-record as a contributor, one of our Maintainers or PMC members may invite you to be a committer (after we've called a vote). When that happens, if you accept, the following process kicks into place...
-
-Note that becoming a committer is not just about submitting some patches; it‘s also about helping out on the development and user [Discussion Forums](https://groups.google.com/forum/#!forum/tidb-user), helping with documentation and the issues.
-
-See [Become a Committer.md](https://github.com/pingcap/community/blob/master/become-a-committer.md) for becoming a committer steps and more details.
-
-## Communication
-
-Communication within the TiDB community abides by [TiDB Code of Conduct](./CODE_OF_CONDUCT.md). Here is an excerpt:
-
-> In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, sex characteristics, gender identity and expression,
-level of experience, education, socio-economic status, nationality, personal
-appearance, race, religion, or sexual identity and orientation.
-
-## Resources
-
-### TiDB Blog
-
-The TiDB Blog is a great resource pool to find content! We post the latest TiDB news including new releases and technical articles there. Check out this page: [https://pingcap.com/blog](https://pingcap.com/blog)
-
-### TiDB Weekly
-
-This weekly newsletter highlights content from the community along with the latest TiDB weekly update & news.
-
-Read the TiDB Weekly archives to catch up on all of the latest TiDB updates and hacks: [https://pingcap.com/weekly](https://pingcap.com/weekly)
-
-### Slideshare
-
-Another great resource for finding content! TiDB’s Slideshare account has slide decks from TiDB presentations -- you’re welcome to use and/or modify these slides when presenting to your own meetup group.
-
-### Tutorial Content
-
-If you're interested in writing a tutorial for TiDB, please contact us: [community@pingcap.com](mailto:community@pingcap.com)
+If you're interested in writing a tutorial for TiDB, please contact us:
```

Comment:
This section feels strange - we don't offer anything tutorial, but instead, we ask for contribution....

Reply:
Yeah, I think we should remove it from this file.